pepper/test/e2e/trade_execution.json

```diff
@@ -0,0 +1,44 @@
+{
+    "tokens": [
+        "T01", 
+        "T02", 
+        "T03"
+    ], 
+    "prices": {
+        "T01": "1000", 
+        "T02": "2000", 
+        "T03": "10000"
+    }, 
+    "orders": [
```

Comment:
Should orders contain some information about accounts?

Reply:
@tomtiger87 is working on that.